Documentation/install/system_requirements.rst

```diff
@@ -136,6 +147,26 @@ by adding the following to the helm configuration command line:
      ...
      --set global.enableXTSocketFallback=false
 
+Advanced Features and Required Kernel Version
+=============================================
+Cilium requires Linux Kernel 4.9.17 or higher, however development on additional
```

Comment:
Nit: `Linux kernel` has a lowercase `k` elsewhere in the documentation.

Reply:
lowercase it is